trimble-oss/modus-web-components
# Table: Set table-layout to fixed for full width table

## Description

Changed table-layout to fixed, as table-layout: auto ignores the column (max-)width.

Before:
![image](https://github.com/trimble-oss/modus-web-components/assets/116563683/f3cf2455-a697-4e05-b316-d5506dab29d2)

After:
![image](https://github.com/trimble-oss/modus-web-components/assets/116563683/7bc85d7e-c9d9-4e19-a6ca-b9a12a128e38)


References #1995

## Type of change


## How Has This Been Tested?

Tested by running in Stencil:
- Create a table
- Set fullWidth: true
- Create a column with a max-width
- Add a value that is longer than the max-width
- Check if result value is concatenated.

## Checklist
- [x] I have made corresponding changes to the documentation _(no docs updated needed)_
